scripts/legislation/africa/lesotho_english.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^\Enacted by the Parliament of Lesotho\./,$p' | \
```

Comment:
Does the `E` in `Enacted` need to be escaped with a backslash?

Reply:
No it does not, thanks for pointing that out.